packages/contracts-bedrock/contracts/universal/OptimismMintableTokenFactory.sol

```diff
@@ -41,15 +41,12 @@ contract OptimismMintableTokenFactory {
     /**
      * @notice Address of the StandardBridge on this chain.
      */
-    address public bridge;
```

Comment:
This contract will technically be upgradable as its a predeploy

Reply:
Yeah I'll put it behind Initializable (along with every other predeploy) in an upcoming PR... we'll need to figure out how that impacts storage layout and if there's any easy way to fix that :-/

Reply:
FWIW, the only predeploy that has a constructor is `GasPriceOracle`, so I don't think the other ones will need `Initializable`?

Though it would be good to add `uint256 constant public version` to them.

Reply:
Note my comment here regarding versioning: https://github.com/ethereum-optimism/optimism/pull/2942#discussion_r915135867